extension/inpage/package.json

```diff
@@ -1,11 +1,11 @@
 {
 	"type": "module",
 	"devDependencies": {
-		"@types/node": "16.11.7",
+		"@types/node": "18.11.12",
```

Comment:
```suggestion
```
`inpage` shouldn't have a dependency on `@types/node` as it is a browser targeted project.

Reply:
It's node project that compiles JavaScript files for the main project

Reply:
Oh, I see.  Ignore this one then.  Still should remove `@node/types` from the extension itself though.